templates/editorial/footer.html

```diff
@@ -1,14 +1,61 @@
+                        </li>
+                        <li>
+                            <a href="{% url 'abous-us' %}" class="fr-footer__top-link">{% translate "Qui sommes-nous" %}</a>
+                        </li>
+                        <li>
+                            <a href="{% url 'partenaires' %}" class="fr-footer__top-link">{% translate "Partenaires" %}</a>
+                        </li>
+                        {# TODO: Uncomment once the FAQ page has been implemented #}
+                        {#                        <li>#}
+                        {#                            <a class="fr-footer__top-link" href="#">{% translate "Une question ?" %}</a>#}
+                        {#                        </li>#}
+                        <li>
+                            <a class="fr-footer__top-link" href="{% url 'contact_form' %}">{% translate "Contactez nous" %}</a>
+                        </li>
+                        <li>
+                            <a class="fr-footer__top-link" href="{% url "challenges" %}">{% translate "Challenges" %}</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="fr-col-12 fr-col-sm-3 fr-col-md-2">
+                    <p class="fr-footer__top-cat">{% translate "Liens externes" %}</p>
+                    <ul class="fr-footer__top-list">
+                        <li>
+                            <a class="fr-footer__top-link"
+                               href="https://www.data.gouv.fr/fr/datasets/accessibilite-des-etablissements-recevant-du-public-erp-pour-les-personnes-en-situation-de-handicap/"
+                               target="_blank">{% translate "Jeu de données" %}</a>
+                        </li>
+                        <li>
+                            <a class="fr-footer__top-link" href="{% url 'apidocs' %}">{% translate "API" %}</a>
+                        </li>
+                        <li>
+                            <a class="fr-footer__top-link" href="{% url 'stats_home' %}">{% translate "Statistiques" %}</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
     <div class="fr-container">
         <div class="fr-footer__body">
             <div class="fr-footer__brand fr-enlarge-link">
-                <a class="d-flex align-items-center text-decoration-none"
-                   href="{% url "home" %}">
-                    <img src="{% static 'img/logo-acceslibre.svg' %}"
-                         alt="{% translate "acceslibre, République Française" %}"
-                         height="75">
+                <a href="{% url "home" %}"
+                    title="{% translate "Retour à l'accueil du site - AccesLibre" %}">
+                    <p class="fr-logo">
+                        <span class="display--block">République</span>
```

Comment:
Missing translation here and on the above line

Reply:
Seen in private, we leave it as is, same for header